tailwind.config.js

```diff
@@ -1,10 +1,24 @@
 module.exports = {
   plugins: [require('tailwindcss')],
   purge: {
-    content: [
-      './docs/.vuepress/**/*.js',
-      './docs/.vuepress/**/*.vue',
-      './docs/.vuepress/**/*.ts',
-    ],
+    content: ['./docs/.vuepress/**/*.js', './docs/.vuepress/**/*.vue', './docs/.vuepress/**/*.ts'],
+  },
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    fill: (theme) => ({
+      red: theme('colors.red.700'),
+      green: theme('colors.green.700'),
+      blue: theme('colors.blue.700'),
```

Comment:
As I observed, we don't use green and red colours on the website. I think we can remove those two attributes.